### PR TITLE
[ADD] point_of_sale: posbox documentation

### PR DESCRIPTION
**Pull request by jorenvo**
This is a big update of the posbox documentation in
addons/hw_proxy/doc. The main reason for this big change is v13 of the
posbox image which is vastly different from all the images that came
before it. On top of that, the documentation was also further updated to
look a bit better when rendered to HTML instead of PDF, which is what we
used before.

It still needs some kind of Point of Sale banner image.

After this is merged, we can get rid of the docs at addons/hw_proxy/doc and
refer to odoo.com/documentation/user from nightly.odoo.com/master/posbox
and from the posbox itself.

### CODE REVIEW

File: point_of_sale/posbox/index.rst [hunk 1]
```diff
@@ -0,0 +1,538 @@
...
+
+* The Posbox
+* A 2A Power adapter
+* A computer or tablet with an up-to-date web browser
+* A running SaaS or Odoo instance with the Point of Sale installed
+* A local network set up with DHCP (this is the default setting)
+* An RJ45 Ethernet Cable or a Linux compatible USB Wi-Fi adapter
+* An Epson USB TM-T20 Printer or another compatible printer
+  (officially supported printers are listed at
+  ``https://www.odoo.com/page/pos-ipad-android-hardware``)
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: replace by a link ````my link <http>`_```

File: point_of_sale/posbox/index.rst [hunk 2]
```diff
@@ -0,0 +1,538 @@
...
+* A Honeywell Eclipse USB Barcode Scanner or another compatible scanner
+* An Epson compatible cash drawer
+
+Step By Step Setup Guide
+------------------------
+
+Current version of the posbox
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: what do you mean by current/old? please specify year or model

File: point_of_sale/posbox/index.rst [hunk 3]
```diff
@@ -0,0 +1,538 @@
...
+
+Old version of the posbox
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. figure:: media/posbox_doc_schema.svg
+
+Connect peripheral devices
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Officially supported hardware is listed on
+``https://www.odoo.com/page/pos-ipad-android-hardware``, but other
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: same as above

File: point_of_sale/posbox/index.rst [hunk 4]
```diff
@@ -0,0 +1,538 @@
...
+hardware might work as well.
+
+Printer
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: can probably be replaced by a list 
```
* **Printer**: Connect...
* **Cash drawer**: ...
```

File: point_of_sale/posbox/index.rst [hunk 5]
```diff
@@ -0,0 +1,538 @@
...
+Once powered, The posbox needs a while to boot. Once the Posbox is
+ready, it should print a status receipt with its IP address. Also the
+status LED, just next to the red power LED, should be permanently lit
+green.
+
+Setup the Point of Sale
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Make sure to activate the right Posbox related options in the Point of
+Sale configuration. Specifying the IP of the Posbox is recommended (it
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: Please detail which options are needed. Also specify where (```:menuselection:`Point of Sale --> Settings --> ....````) or add a screenshot

File: point_of_sale/posbox/index.rst [hunk 6]
```diff
@@ -0,0 +1,538 @@
...
+
+The goal of this section will be to setup a local Odoo instance that
+behaves like the Odoo instance running on the Posbox.
+
+Image building process
+----------------------
+
+We generate the posbox images at
+``http://nightly.odoo.com/master/posbox/`` using the scripts in
+``https://github.com/odoo/odoo/tree/8.0/addons/point_of_sale/tools/posbox``. More
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: 8.0 scripts ? shouldn't be 9.0 ?

File: point_of_sale/posbox/index.rst [hunk 7]
```diff
@@ -0,0 +1,538 @@
...
+called ``posbox.img``, which we zip and upload to ``nightly.odoo.com``
+for users to download.
+
+The scripts in this directory might be useful as a reference if you
+get stuck or want more detail about something.
+
+Summary of the image creation process
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The image creation process starts by downloading the latest Raspbian
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: link to rapsbian software

File: point_of_sale/posbox/index.rst (hunk 6 above)
Comment (jorenvo) [inline point_of_sale/posbox/index.rst]: No, all the posbox stuff is still in 8.0. Although part of the posbox stuff is forward-ported I think. But we still use 8.0 for the official images.

File: point_of_sale/posbox/index.rst [hunk 8]
```diff
@@ -0,0 +1,538 @@
...
+Then you need to reboot your machine.
+
+Start the local Odoo instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We must launch the Odoo server with the correct settings
+
+``$ ./odoo.py --load=web,hw_proxy,hw_posbox_homepage,\``
+
+``hw_posbox_upgrade,hw_scale,hw_scanner,hw_escpos``
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: no need for multiline

File: point_of_sale/posbox/index.rst [hunk 9]
```diff
@@ -0,0 +1,538 @@
...
+Posbox Technical Documentation
+==============================
+
+Technical Overview
+------------------
+
+The Posbox Hardware
+~~~~~~~~~~~~~~~~~~~
+
+The Posbox's Hardware is based on a Raspberry Pi 2, a popular Open
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: link to RP website

File: point_of_sale/posbox/index.rst [hunk 10]
```diff
@@ -0,0 +1,538 @@
...
+~~~~~~~~~~~~~~~~~~~
+
+The Posbox runs a heavily modified Raspbian Linux installation, a
+Debian derivative for the Raspberry Pi. It also runs a barebones
+installation of Odoo which provides the webserver and the drivers. The
+hardware drivers are implemented as Odoo modules. Those modules are
+all prefixed with ``hw_*`` and they are the only modules that are
+running on the Posbox. Odoo is only used for the framework it
+provides. No business data is processed or stored on the Posbox. The
+Odoo instance is a shallow git clone of the ``8.0`` branch.
```
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: again 8.0 ? the doc is based on 9.0

File: point_of_sale/posbox/index.rst (hunk 6 above)
Comment (mart-e) [inline point_of_sale/posbox/index.rst]: ok that we need to sort it out, would be good to port everything to 9 as all the user doc assume we are using 9.0

Comment (jorenvo) [inline point_of_sale/posbox/index.rst]: It's just the posbox that runs 8.0 internally. It should eventually run 9.0, but it's really not that important.

**Comment by mart-e** [comment]
and a banner for POS section :)
pos

**Comment by photoscene** [comment]
isn’t it generated based on the same image?
It’s wide enough for the banner, but if you need a different image just tell me :)


--
Csaba Fazekas
webdesigner

Chaussée de Namur, 40
B-1367 Grand-Rosière




odoo.com  <http://www.odoo.com/>| Facebook | <https://www.facebook.com/odoo> Twitter | <https://twitter.com/Odoo> Linkedin | <https://www.linkedin.com/company/odoo> Google Plus | <https://plus.google.com/u/0/b/100883176139028080672/100883176139028080672/posts> Youtube <https://www.youtube.com/user/OpenERPonline>

**Comment by mart-e** [comment]
@photoscene no thanks, it's fine (just needed to convert it to jpg). Will add it at the merge.

**Comment by mart-e** [comment]
@jorenvo posbox, Posbox or PosBox ? :-) could be nice to harmonize to one

**Comment by mart-e** [comment]
merged at 200aa1389e2e82c9e7c6b9658e230ca69c69bf76 thanks for your work!
(we can always improve afterwards anyway)

**Pull request closed without merging**